sickbeard/tv.py

```diff
@@ -500,25 +501,26 @@ def loadEpisodesFromDB(self):
 
         for curResult in sqlResults:
 
-            logger.log(u"loadEpisodesFromDB curResult: " + str(curResult), logger.DEBUG)
-            deleteEp = False
-
             curSeason = int(curResult["season"])
             curEpisode = int(curResult["episode"])
+            curShowid = int(curResult['showid'])
+
+            logger.log(u"%s: loading Episodes from DB" % curShowid, logger.DEBUG)
```

Comment:
@miigotu this was returning empty in log: str(curResult)

I created curShowid = int(curResult['showid'])

Or show do I show all items in curResult?